src/types/CountOfCasesByCaseAgeResult.ts

```diff
@@ -0,0 +1,6 @@
+export type CountOfCasesByCaseAgeResult = {
```

Comment:
Since we already have a `CaseAge` type, I wonder whether we could use a more generic type like `{[key: CaseAge]: number}` instead of duplicating all the case age options here